specs/select.spec.md

```diff
@@ -0,0 +1,158 @@
+# Select + Select-Option
```

Comment:
I think there should be some documentation around proper usage of a Select inside or outside of a `<form>`

Reply:
DIdn't see this previously. As select is a form element I think guidance san simply be that we recommend only using it within a form.